titus-server-runtime/src/main/java/com/netflix/titus/runtime/kubernetes/KubeConstants.java

```diff
@@ -179,4 +181,20 @@ private KubeConstants() {
      * Reconciler Event Constants
      */
     public static final String NODE_LOST = "NodeLost";
+
+    /**
+     * Volume annotations.
+     */
+    public static final String VOLUME_DOMAIN = "volume." + NETFLIX_DOMAIN;
+    public static final String EBS_DOMAIN = "ebs." + VOLUME_DOMAIN;
+    public static final String EBS_VOLUME_ID = EBS_DOMAIN + "volume-id";
```

Comment:
camelcase on these?
https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects

Reply:
They were camelcase and @rgulewich mentioned the annotation style being used it lower case, dash separated (https://github.com/Netflix/titus-kube-common/blob/pod-spec/pod/annotations.go).

Reply:
The k8s docs don't explicitly say not to use camelcase, but all of the k8s annotations themselves use lowercase plus dashes, so we should try to be consistent: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

Reply:
Heh, got it.